sys-boot/refind/refind-0.14.0.2-r1.ebuild

```diff
@@ -0,0 +1,183 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit toolchain-funcs secureboot
+
+DESCRIPTION="The UEFI Boot Manager by Rod Smith"
+HOMEPAGE="https://www.rodsbooks.com/refind/"
+SRC_URI="mirror://sourceforge/project/${PN}/${PV}/${PN}-src-${PV}.tar.gz"
+
+LICENSE="BSD GPL-2 GPL-3 FDL-1.3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+FS_USE="btrfs +ext2 +ext4 hfs +iso9660 ntfs reiserfs"
+IUSE="${FS_USE} custom-cflags doc"
+
+DEPEND="sys-boot/gnu-efi"
+
+# for ld.bfd and objcopy
+BDEPEND="sys-devel/binutils"
+
+DOCS=( README.txt NEWS.txt )
+
+PATCHES=( "${FILESDIR}"/${P}-clang.patch )
+
+checktools() {
+	if [[ ${MERGE_TYPE} != "binary" ]]; then
+		# 832018
+		export LD="$(tc-getLD)"
+		tc-ld-force-bfd
+		# tc-ld-force-bfd does not work in all cases, so try to also fix LD
```

Comment:
you sure?

Reply:
Yes, I tried.  This makefile does not use clang or gcc to link, it calls LD directly.  I looked in the eclass and it exports LD sometimes, but not always.  The heuristic I tried below works for the case in ago's bug, but in case it doesn't I added a check with || die.

Reply:
OK, please add a comment saying it calls LD directly, hence the need for the hack.